src/main/java/proj/kedabra/billsnap/business/facade/NotificationFacade.java

```diff
@@ -0,0 +1,10 @@
+package proj.kedabra.billsnap.business.facade;
+
+import proj.kedabra.billsnap.business.dto.AnswerNotificationDTO;
+import proj.kedabra.billsnap.business.dto.BillSplitDTO;
+
+public interface NotificationFacade {
+
+    BillSplitDTO answerInvitation(AnswerNotificationDTO answerNotificationDTO);
```

Comment:
Now that I think about it, if we decline the invitation, there's no need for the user to see the bill or even have access to the bill. If that's the case, would you be returning null? 

Otherwise, would we need to change the spec so that we return another type of resource?

Reply:
That's a good point. We could indeed change spec to return null in the case of decline

Reply:
Sure, idm. Just note that the controller will have to take that into consideration as to not map a null value